Book/php7/extensions_design/hooks.rst

```diff
@@ -88,11 +88,20 @@ calls.::
         original_handler_var_dump(INTERNAL_FUNCTION_PARAM_PASSTHRU);
     }
 
+    // If the ZEND_TSRMLS_CACHE_UPDATE() is in RINIT, move it
+    // to GINIT to ensure access to compiler globals in MINIT
```

Comment:
Shouldn't it be part of MINIT instead?

Reply:
Ping. I don't see why this should be in GINIT rather than MINIT.

Reply:
@nikic Sorry for the radio silence. I've moved this to MINIT.